src/igvfd/types/file_set.py

```diff
@@ -118,6 +118,73 @@ class CuratedSet(FileSet):
     schema = load_schema('igvfd:schemas/curated_set.json')
     embedded_with_frame = FileSet.embedded_with_frame
 
+    @calculated_property(
+        define=True,
+        schema={
+            'title': 'Assembly',
+            'description': 'The genome assembly to which the referencing files in the file set are utilizing (e.g., GRCh38).',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'type': 'string'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def assembly(self, request, files):
+        if files is not None:
+            assembly_values = set()
+            for current_file_path in files:
+                file_object = request.embed(current_file_path, '@@object?skip_calculated=true')
+                if file_object.get('assembly'):
+                    assembly_values.add(file_object.get('assembly'))
+        if assembly_values:
+            return list(assembly_values)
+
+    @calculated_property(
+        define=True,
+        schema={
+            'title': 'Transcriptome Annotation',
+            'description': 'The annotation and version of the reference resource.',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'type': 'string'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def transcriptome_annotation(self, request, files):
+        if files is not None:
+            annotation_values = set()
+            for current_file_path in files:
+                file_object = request.embed(current_file_path, '@@object?skip_calculated=true')
+                if file_object.get('transcriptome_annotation'):
+                    annotation_values.add(file_object.get('transcriptome_annotation'))
+        if annotation_values:
+            return list(annotation_values)
+
+    @calculated_property(
+        schema={
+            'title': 'Summary',
+            'type': 'string',
+            'notSubmittable': True,
+        }
+    )
+    def summary(self, curated_set_type, assembly=None, transcriptome_annotation=None, taxa=None):
+        summary_message = curated_set_type
+        if taxa:
+            summary_message += f' {taxa}'
+        if assembly:
+            assembly.sort()
```

Comment:
why do you need to sort?

Reply:
I believe Kalina would have implemented sorting to ensure during testing that the summary has the assemblies listed in the order she asserted. (I guess? - I've done the same thing with my sample summary ticket)

I don't think there's harm in doing so and it will make the summary look cleaner to have assemblies listed in alphabetical order.

Reply:
Personally, I wish more potentially long lists were alphabetized. It makes it easier to do a binary search with your eyes.

Reply:
agree that need to be sorted for summary consistency